Source/Scene/PointCloudPostProcessor.js

```diff
@@ -95,65 +110,138 @@ define([
                 pixelDatatype : PixelDatatype.UNSIGNED_INT_24_8,
                 sampler : createSampler()
             });
+        }
 
-            // TODO : for now assume than any pass can write depth, possibly through EXT_frag_depth.
-            // Definitely needed for the initial render into the FBO, possibly also the ping-pong processing.
-            framebuffers[i] = new Framebuffer({
+        /* We want to reuse textures as much as possible, so here's the order
+         * of events:
+         *
+         * 1.  We render normally to our prior:
+         *     * Color -> 0
+         *     * Depth -> 2 ("dirty depth")
+         *     * EC -> 0
+         * 2.  Then we perform the screen-space point occlusion stage with color[0] and EC:
+         *     * No color
+         *     * Depth -> 0
+         *     * No EC
+         * 3a. Then we perform the region growing stage with color[0] and depth[0]:
+         *     * Color -> 1
+         *     * Depth -> 1
+         * 3b. We do the region growing stage again with color[1] and depth[1]:
+         *     * Color -> 0
+         *     * Depth -> 0
+         * 3c. Repeat steps 3a and 3b until all holes are filled and/or we run
+         *     out of time.
+         */
+
+        processor._framebuffers = {
+            "prior": new Framebuffer({
```

Comment:
Here and similar properties below, `"prior"` -> `prior`

Quotes are not needed here.  When they are, use single quotes to be consistent with the rest of the code base.